close 1078 option cleanup (#1109)
exercism/rust: concepts/option/introduction.md

> The standard library provides an enum called `Option` to help you express when a value exists or does not exist.

Reviewer: ```suggestion
The standard library provides an enum called `Option` to help you express when a value may or may not exist.
```

Not mandatory; I just think this way sounds better.
